docs/reference/permission.rst

```diff
@@ -1,26 +1,125 @@
 .. _permission:
 
-Permission
-##########
+Permissions
+###########
 
+*Cliquet* provides a mechanism to handle authorization on the stored objects.
 
-Redis
-=====
+Glossary
+========
 
-.. autoclass:: cliquet.permission.redis.Redis
+Authorization isn't complicated, but requires the introduction of a few terms
+so that explanations are easier to follow:
 
+**Object**:
```

Comment:
I thought about that but I'm actually not sure, since it really is a prerequisite to understand this section. Maybe duplicating is okay, or maybe there is a way to use the definitions that are stored there.